content/millennium/r4/devices.md

```diff
@@ -4,4 +4,4 @@ layout: api
 ---
 
 <%= render '/devices.*' %>
-* [Device](../devices/device)
\ No newline at end of file
+* [Device](../devices/device)
```

Comment:
can you explain what changed here?

Reply:
It was unintentional.  I [reverted](https://github.com/cerner/fhir.cerner.com/pull/334/commits/5e38b77a2b8751100b9950d56d54324d1e102e2b#diff-6f15a88327bf5b0d1741ad3afe843e6eR7) to the last commit's version of the file.